homeassistant/components/aws/const.py

```diff
@@ -19,12 +15,15 @@
 CONF_SECRET_ACCESS_KEY = "aws_secret_access_key"
 CONF_SERVICE = "service"
 CONF_VALIDATE = "validate"
-
 CONF_COLLECTION_ID = "collection_id"
 CONF_IDENTIFY_FACES = "identify_faces"
 CONF_DETECTION_ATTRIBUTES = "detection_attributes"
 CONF_SAVE_FILE_FOLDER = "save_file_folder"
 CONF_SAVE_FILE_TIMESTAMP = "save_file_timestamp"
 
-EVENT_OBJECT_DETECTED = "image_processing.detect_object"
-EVENT_LABEL_DETECTED = "image_processing.detect_label"
+DATA_CONFIG = "aws_config"
+DATA_HASS_CONFIG = "aws_hass_config"
+DATA_SESSIONS = "aws_sessions"
+
+EVENT_OBJECT_DETECTED = "rekognition.detect_object"
```

Comment:
in my variant I am using object_detected which I feel is a more natural description. However I realise you are being consistent with detect_face

Reply:
Completely agree `object_detected` and `face_detected` is much more natural, but with `detect_face` coming from the base component I think consistency wins in this scenario

Reply:
The expression 'two wrongs do not make a right' comes to mind :)
In this case a breaking change is inevitable for some users:

1) Use of `detect_object` is a breaking change for the users of [my custom integration](https://github.com/robmarkcole/HASS-amazon-rekognition),. Also its then incompatible with my [deepstack integration](https://github.com/robmarkcole/HASS-Deepstack-object) which I plan on making official soon. Combining stars on those two projects estimate 200 users affected.
2) Use `object_detected` and update the face integration to use `face_detected`. Cannot estimate how many users this affects. **Note**, the `detect_face` event was [subject to a breaking change](https://community.home-assistant.io/t/0-38-alert-appletv-mqtt-discovery-and-yeelight/42558), as was originally named `identify_face`. 

My personal preference is for (2), but suggest we get another review in on this point, perhaps @pvizeli

Reply:
I'll keep it as-is for now to await further feedback. I think you make a really good point around the existing users of your custom integration - it's absolutely the gold standard and this implementation should follow it's structure as much as possible especially given the active user-base.